core/templates/pages/contact-page/contact-page.component.html

```diff
@@ -4,35 +4,35 @@ <h1 tabindex="0" [innerHTML]="'I18N_CONTACT_PAGE_HEADING' | translate"></h1>
 <background-banner></background-banner>
 <div class="oppia-page-card oppia-static-full-content oppia-static-content-contact-page-below-banner">
   <div class="oppia-static-card-full-content oppia-static-card-content-full-wide e2e-test-contact-page">
-    <h2 [innerHTML]="'I18N_CONTACT_PAGE_GREETING_HEADING' | translate"></h2>
+    <h2 [innerHTML]="'I18N_CONTACT_PAGE_GREETING_HEADING' | translate" tabindex="0"></h2>
 
     <div class="oppia-contact-page flex">
-      <div class="oppia-contact-page-content">
-        <img id="donate" src="assets/images/contact_page/Donate img.svg" alt="Donate">
+      <div class="oppia-contact-page-content" tabindex="0">
+        <img id="donate" src="assets/images/contact_page/Donate img.svg" alt="">
         <h2 [innerHTML]="'I18N_CONTACT_PAGE_DONATE_HEADING' | translate"></h2>
         <p [innerHTML]="'I18N_CONTACT_PAGE_DONATE_PARAGRAPH' | translate"></p>
         <a href="https://www.oppia.org/donate">
-          <button class="hover-button" [innerHTML]="'I18N_CONTACT_PAGE_DONATE_TODAY_BUTTON' | translate"></button>
+          <button tabindex="-1" class="hover-button" [innerHTML]="'I18N_CONTACT_PAGE_DONATE_TODAY_BUTTON' | translate"></button>
         </a>
       </div>
-      <div class="oppia-contact-page-content">
-        <img id="Partner" src="assets/images/contact_page/Partner img.svg" alt="Partner">
+      <div class="oppia-contact-page-content" tabindex="0">
+        <img id="Partner" src="assets/images/contact_page/Partner img.svg" alt="">
         <h2 [innerHTML]="'I18N_CONTACT_PAGE_PARTNER_HEADING' | translate"></h2>
         <p [innerHTML]="'I18N_CONTACT_PAGE_PARTNER_PARAGRAPH' | translate"></p>
         <a href="https://www.oppia.org/partnership">
-          <button class="hover-button" [innerHTML]="'I18N_CONTACT_PAGE_BECOME_A_PARTNER_BUTTON' | translate"></button>
+          <button class="hover-button" tabindex="-1" [innerHTML]="'I18N_CONTACT_PAGE_BECOME_A_PARTNER_BUTTON' | translate"></button>
         </a>
       </div>
-      <div class="oppia-contact-page-content">
-        <img id="volunteer" src="assets/images/contact_page/Volunteer img.svg" alt="Volunteer">
+      <div class="oppia-contact-page-content" tabindex="0">
+        <img id="volunteer" src="assets/images/contact_page/Volunteer img.svg" alt="">
         <h2 [innerHTML]="'I18N_CONTACT_PAGE_VOLUNTEER_HEADING' | translate"></h2>
         <p [innerHTML]="'I18N_CONTACT_PAGE_VOLUNTEER_PARAGRAPH' | translate"></p>
         <a href="https://www.oppia.org/volunteer">
-          <button class="hover-button" [innerHTML]="'I18N_CONTACT_PAGE_BECOME_A_VOLUNTEER_BUTTON' | translate"></button>
+          <button class="hover-button" tabindex="-1" [innerHTML]="'I18N_CONTACT_PAGE_BECOME_A_VOLUNTEER_BUTTON' | translate"></button>
         </a>
       </div>
-      <div class="oppia-contact-page-content">
-        <img id="other_inquiries" src="assets/images/contact_page/Other inquiries img.svg" alt="Other inquiries">
```

Comment:
Any reason for removing the alt text?

Reply:
The alt text is not conveying any relevant useful information to the user as this is for the thumbnail, for @Rijuta-s had suggested me earlier for similar case senario to remove the alt text and keep it blank.